src/sub-blocks/BasicCard/BasicCard.tsx

```diff
@@ -23,33 +22,46 @@ const BasicCard = (props: BasicCardProps) => {
         list,
         buttons,
         iconPosition = IconPosition.Top,
+        controlPosition = 'content',
         ...cardParams
     } = props;
     const titleId = useUniqId();
     const descriptionId = useUniqId();
+    const areControlsInFooter = controlPosition === 'footer';
 
     return (
         <CardBase
             className={b()}
             {...cardParams}
             extraProps={{'aria-describedby': descriptionId, 'aria-labelledby': titleId}}
         >
-            <CardBase.Content>
+            <CardBase.Content key="content">
                 <IconWrapper icon={icon ? {value: icon, position: iconPosition} : undefined}>
                     <Content
                         title={title}
                         titleId={titleId}
                         text={text}
                         textId={descriptionId}
                         additionalInfo={additionalInfo}
-                        links={links}
+                        links={areControlsInFooter ? undefined : links}
                         list={list}
-                        buttons={buttons}
+                        buttons={areControlsInFooter ? undefined : buttons}
                         size="s"
                         colSizes={{all: 12, md: 12}}
                     />
                 </IconWrapper>
             </CardBase.Content>
+            {areControlsInFooter && (links || buttons) && (
```

Comment:
this code repeats 3 times in pr, could we create kinda component?

Reply:
As I wrote earlier, it's impossible to wrap `CardBase.Footer` in other components. Because the `CardBase` component uses its children's types within itself. Therefore I think that we should not update code here.